pkg/volume/flocker/flocker.go

```diff
@@ -203,7 +203,7 @@ func (b *flockerVolume) GetDatasetUUID() (datasetUUID string, err error) {
 	}
 
 	if b.flockerClient == nil {
-		return "", fmt.Errorf("Flocker client is not initialized")
+		return "", fmt.Errorf("flocker client is not initialized")
```

Comment:
think "Flocker" can be capitalized as a proper noun

Reply:
ok, got it.